pkg/handler/handler.go

```diff
@@ -149,18 +147,18 @@ func logContext(podName, podGenerateName, serviceAccountName, namespace string)
 	if len(podName) == 0 {
 		name = podGenerateName
 	}
-	return fmt.Sprintf("Pod=%s, " +
-		"ServiceAccount=%s, " +
+	return fmt.Sprintf("Pod=%s, "+
+		"ServiceAccount=%s, "+
 		"Namespace=%s",
 		name,
 		serviceAccountName,
 		namespace)
 }
 
-func (m *Modifier) addEnvToContainer(container *corev1.Container, tokenFilePath, roleName string, podSettings *podUpdateSettings) {
+func (m *Modifier) addEnvToContainer(container *corev1.Container, tokenFilePath, roleName string, podSettings *podUpdateSettings) bool {
 	// return if this is a named skipped container
 	if _, ok := podSettings.skipContainers[container.Name]; ok {
-		return
+		return false
 	}
```

Comment:
Should we log in these cases?

Reply:
Ya. We can add a log here as well. Will fix

Reply:
Updated